install_sct

```diff
@@ -57,6 +89,19 @@ for installation instructions check: https://git-scm.com/book/en/v2/Getting-Star
 "
   fi
 
+  if [[ ! $(which gcc) ]]; then
+    if [[ $OS == "osx" ]]; then
+      read -p 'You need to install gcc before you can use the SPINALCORDTOOLBOX. Do you want to install gcc now? [y]es/[n]o: ' -r GCC_INSTALL
+      if [[ $GCC_INSTALL =~ [Yy](es)? ]]; then
+        brew install gcc
```

Comment:
`brew` is not installed natively on OSX (unfortunately), so this could break

Reply:
Dang! 😢 

I'll remove that part.

Reply:
one possibility would be to add another condition that would check if brew is installed, and if not, [installs it](https://brew.sh/)

Reply:
Or I could add a check to see if the user has brew installed.